### ISSUE TITLE
Modded Consumption

### ISSUE THREAD

**Issue by tmanfoo1** [EdwinMindcraft/origins-architectury#134]
[blood_bucket1.txt](https://github.com/EdwinMindcraft/origins-architectury/files/8394651/blood_bucket1.txt)
[blood_bottle.txt](https://github.com/EdwinMindcraft/origins-architectury/files/8394652/blood_bottle.txt)
[blood_bucket2.txt](https://github.com/EdwinMindcraft/origins-architectury/files/8394653/blood_bucket2.txt)
[blood_slime_balls.txt](https://github.com/EdwinMindcraft/origins-architectury/files/8394654/blood_slime_balls.txt)
These are in .txt format so I could upload them here. 
Long story short they make eating noises, but dont replentish health or consume the item

**Comment by EdwinMindcraft** [comment]
Change your checks from `origins:food_level` to [`origins:health`](https://origins.readthedocs.io/en/latest/types/entity_condition_types/health/) or [`origins:relative_health`](https://origins.readthedocs.io/en/latest/types/entity_condition_types/relative_health/)
If I remember correctly, those origins cannot get below 20 food level.
